src/main/scala/dxWDL/base/WomMetaSerialization.scala

```diff
@@ -0,0 +1,20 @@
+package dxWDL.base
+
+import wom.callable.MetaValueElement
+import wom.callable.MetaValueElement._
+
+// Convert WOM MetaValueElement to/from JSON, etc
+object WomMetaSerialization {
+  def unwrapAny(element: MetaValueElement): Any = {
```

Comment:
It is possible to use the `Any` type, but it is preferable to use a specific type. Further down the line the code calls `anyToJs` converting the result value into a `JsValue`. Could you just convert it right here to a `JsValue`? There may be other solutions. The issue with `Any` is that when you get a value of `Any` type you are using dynamic typing; you don't know at compile time what the type is. This significantly reduces the utility of static type checking.

Reply:
This is specifically for unwrapping details, which is an object with arbitrary keys and values. I would opt to leave it as a MetaValueElement rather than covert it to Js representation in IR.

Reply:
Sure, that works. `MetaValueElement` is more specific than `Any`.

Reply:
Ok that's done